lib/tasks/build_artifact.rake

```diff
@@ -15,8 +15,8 @@ task :build_artifact do |task|
   FileUtils.mkdir_p('pkg')
   FileUtils.mkdir_p('cache')
   FileUtils.mkdir_p('session')
-  FileUtils.mkdir_p('log')
   FileUtils.touch('config.php')
+  FileUtils.rm('log/.gitignore')
```

Comment:
@willaerk or @paulherbosch since I also edited a `*.rake`-file, I will need one of you to 2 to approve this PR. It's basically the same change as Kristof did in his PR to make it work on Jenkins: https://github.com/cultuurnet/udb3-backend/pull/1800